shimmingtoolbox/coils/coil.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+
+class Coil(object):
+    """
+    Coil profile object that stores coil profiles and there constraints
+
+    Attributes:
+        dim (tuple int): Dimension along specific axis. dim:0,1,2 are spatial axes, while dim:3 corresponds to the coil
+                         channel.
+        profile (numpy.ndarray): (dim1, dim2, dim3, channels) 4d array of N 3d coil profiles
+        affine (np.ndarray): 4x4 array containing the affine transformation for the coil profiles
+        coef_sum_max (float): Contains the maximum value for the sum of the coefficients
+        coef_channel_minmax (list): Contains the maximum coefficient for each channel
+    """
+
+    def __init__(self, profile, affine, constraints):
+        """
+
+        Args:
+            profile (np.ndarray): Coil profile (dim1, dim2, dim3, channels) 4d array of N 3d coil profiles
+            affine (np.ndarray): 4x4 array containing the affine transformation for the coil profiles
```

Comment:
maybe check or specify in the docstring if you are referring to the qform or sform-- we might also want to add a check to make sure that both sform (used by FSL/SPM) and qform (used by ITK/SCT) are the **same**. They should, because no processing (transformation) should be run on the coil profile.